package.json

```diff
@@ -4,8 +4,8 @@
   "description": "JS implementation of the SQL JSONPath dialect, from SQL2016.",
   "main": "dist/index.ts",
   "scripts": {
-    "test": "test",
-    "start": "npx tsc && node ./dist/index.js"
+    "test": "mocha -r ts-node/register test/**/test.ts",
+    "start": "npx tsc && node ./dist/src/index.js"
```

Comment:
Add this to the end of tsconfig.json as a peer to "compilerOptions":

```"include": ["./src"]```

and then `src/` will be hoisted away in `dist/`

Reply:
I get `Unknown compiler option 'include'` when I try that. Does it work for you?